reconcile/openshift_namespaces.py

```diff
@@ -103,7 +103,10 @@ def create_new_project(spec, oc_map):
     cluster = spec['cluster']
     namespace = spec['namespace']
 
-    oc_map.get(cluster).new_project(namespace)
+    oc = oc_map.get(cluster).new_project(namespace)
+    if not oc:
+        logging.log(level=oc.log_level, msg=oc.message)
+        return None
```

Comment:
I meant:
```
    oc = oc_map.get(cluster)
    if not oc:
        logging.log(level=oc.log_level, msg=oc.message)
        return None
    oc.new_project(namespace)
```

Reply:
:+1:

Reply:
`oc_map.get(cluster).new_project(namespace)` does not give you an `OC()`/`OcLogMsg()` instance.

Reply:
Just made this change :+1: